CI/travis.osx.after_success.sh

```diff
@@ -161,6 +217,24 @@ if [ "${DEPLOY}" = "deploy" ]; then
       -F "output=json" \
       -F "do=Add File"
 
+      # Upload portable version
```

Comment:
Need to do the same for Linux

Reply:
Right! But where it stores the data is not in a portable location, it stores it in `$HOME/.config` always, whereas a portable-portable one would store it relative to the AppImage I imagine, so you could have Mudlet on a USB stick.

Reply:
Addressed in new commit